apps/src/templates/PuzzleRatingButtons.jsx

```diff
@@ -57,6 +69,10 @@ class PuzzleRatingButtons extends Component {
     };
   }
 
+  static propTypes = {
+    useLegacyStyles: React.PropTypes.bool,
```

Comment:
The arrow in the target should not have a light purple outline, @joshlory did someone ask you to change it? In my comp it is the same solid purple as everything else, this light purple inline makes it harder to read and kills the design. It also looks like there is a light gray behind the target, anything not purple should be white #fff

I didn't know we were changing the level review buttons, is that documented somewhere?